.github/workflows/ci.yaml

```diff
@@ -40,10 +40,10 @@ jobs:
           - .
           - tests/integration/test-charm-azure
     name: Build charm
-    uses: canonical/data-platform-workflows/.github/workflows/build_charm.yaml@v16.2.1
+    uses: canonical/data-platform-workflows/.github/workflows/build_charm.yaml@v24.0.6 
     with:
       path-to-charm-directory: ${{ matrix.path }}
-      cache: true
+      cache: false # TODO: Add to ccc-hub
```

Comment:
IIRC, this repo already exists in CCC Hub. No?

Reply:
Only the test app is in the hub, this is weird ahah

Reply:
:thinking: yeah, i remember doing it in a PR.. maybe it was reverted at some point unintentionally or intentionally to unblock the CI in the CCC Hub. NVM, this was only a question -- this PR looks super good to me already.